es-ES/websites_tou.md

```diff
@@ -90,22 +90,21 @@ LOS MENSAJES SE FACILITAN "TAL Y COMO ESTÁN", CON TODOS SUS FALLOS. EN LA MEDID
 
 
-#### 11\. Modificaciones de las presentes Condiciones  
+#### 11\. Modificaciones de las presentes Condiciones
 
 Nos reservamos el derecho a actualizar periódicamente las presentes Condiciones con el fin de adecuarlas a nuevas características de los Mensajes o de aclarar alguna disposición. La versión actualizada de las Condiciones se publicará en línea. Si los cambios realizados fuesen sustanciales, anunciaremos la actualización mediante los canales que utilizamos habitualmente para tal fin, como publicaciones en blogs, avisos, correos electrónicos y foros. En caso de que, tras la fecha de entrada en vigor de las modificaciones, siga usando los Mensajes, se entenderá que acepta tales cambios. Para que le resulte más cómodo revisar las presentes Condiciones, publicaremos la fecha de entrada en vigor de las modificaciones en la parte superior de esta página.
 
-#### 12\. Aspectos varios  
+#### 12\. Aspectos varios
 
 Las presentes Condiciones constituyen la totalidad del acuerdo entre Mozilla y usted con respecto al uso de los Mensajes y prevalecen sobre las versiones anteriores de las mismas. Los Mensajes y las presentes Condiciones se rigen por las leyes del estado de California, EE. UU., excluyendo sus disposiciones sobre conflicto de leyes. Todas las reclamaciones y disputas derivadas de los Mensajes o de las presentes Condiciones se presentarán exclusivamente en los tribunales y juzgados ubicados en el condado de Santa Clara, California, y usted acepta someterse a la jurisdicción personal de los mismos. Si alguna parte de las presentes Condiciones se considera no válida o no ejecutable, el resto tendrá plena vigencia y efecto. En caso de que existan discrepancias entre una versión traducida de las presentes Condiciones y la versión en inglés, será esta última la que rija. En el supuesto de que se produzca un conflicto entre las presentes Condiciones y los términos adicionales pertinentes, serán estos últimos los que rijan.
 
 #### 13\. Póngase en contacto con nosotros
 
 Mozilla
-A la atención de: Mozilla – Legal Notices  
-331 E. Evelyn Ave.,  
-Mountain View, CA 94041  
-EE. UU.  
-Teléfono: 650-903-0800  
-Fax: 650-903-0875  
-Avisos legales en mozilla.com
-
+Attn: Mozilla – Legal Notices
+2 Harrison St # 175,
+San Francisco, CA 94105
+USA
+Teléfono: 650-903-0800
+Fax: 650-903-0875
+Legal-notices at mozilla.com
```

Comment:
Per https://daringfireball.net/projects/markdown/syntax
> _"When you do want to insert a `<br />` break tag using Markdown, you **end a line with two or more spaces**, then type return."_

I wonder if this will cause the address to now run on one line.
As close as I can tell, these "[locale]/websites_tou.md" files are used in https://www.mozilla.org/en-US/about/legal/terms/mozilla/, so might be worth confirming the layouts after the pages get redeployed to make sure the addresses are still properly formatted.

Reply:
Shut, it does :-( 

https://www-dev.allizom.org/es-ES/about/legal/terms/mozilla/

I used a different editor for updating the other docs, I really think relying on double spaces for new lines is a terrible idea, because at some point someone will remove them (like I did).

Reply:
This should be fixed by #1670, waiting for automation to pick up the changes

Reply:
> _"I really think relying on double spaces for new lines is a terrible idea"_

💯 , but sadly it's now forever in the Markdown spec.
Although double checking against the Commonmark spec, there might be an alternate syntax option of EOL backslash: https://commonmark.org/help/tutorial/03-paragraphs.html
> _"For a line break, add either a backslash `\`, or two blank spaces at the end of the line."_

But then double-double checking that against the [Markdown dingus](https://daringfireball.net/projects/markdown/dingus), dashes my hopes and dreams yet again.

```md
the quick\
brown fox  
lazy dog
```

### HTML (Markdown)
```html
<p>the quick\
brown fox <br />
lazy dog</p>
```

### HTML (Commonmark)
```html
<p>the quick<br>
brown fox<br>
lazy dog</p>
```

TLDR: Depending on which Markdown engine mozilla.org uses, we might be able to use commonmark trailing slash syntax instead of trailing double spaces [but probably best not to, for compatibility].

Reply:
Also note that Markdown is a superset of HTML and you can just use `<br>` as well. We could test out the backslash though.